lib/ace/mode/rst_highlight_rules.js

```diff
@@ -171,6 +178,66 @@ var RSTHighlightRules = function() {
       }
     ],
 
+    /* Inline code
+     * The parser returns to the "start" state when reaching "``" */
+    "code": [
+      {
+        token : "markup.raw",
```

Comment:
it might be better to use classes which are highlighted by themes, similar to markdown mode
https://github.com/ajaxorg/ace/blob/master/lib/ace/mode/markdown_highlight_rules.js#L106
or maybe better to asciidoc mode
https://github.com/ajaxorg/ace/blob/master/lib/ace/mode/asciidoc_highlight_rules.js#L199